internal/pkg/manifest/testdata/environment-default.yml

```diff
@@ -0,0 +1,21 @@
+# The manifest for the "test" environment.
+# Read the full specification for the "Environment" type at:
+#  https://aws.github.io/copilot-cli/docs/manifest/environment/
+
+# Your environment name will be used in naming your resources like VPC, cluster, etc.
+Name: test
+Type: Environment
+
+# Import your own VPC and subnets or configure how they should be created.
+# network:
+#   vpc:
+#     id:
```

Comment:
This would be so awesome, if we can write eventually
```suggestion
# network:
#   vpc:
#     azs: 3
```

Reply:
agreed! Currently this manifest is a strictly minimal to achieve parity with existing flags, therefore `azs` isn't there yet. After we roll out `vpc.azs` we can update the default commented-out manifest with it!